beacon-chain/rpc/beacon/attestations_test.go

```diff
@@ -1002,7 +1002,7 @@ func TestServer_AttestationPool_Pagination_CustomPageSize(t *testing.T) {
 				PageSize:  int32(numAtts),
 			},
 			res: &ethpb.AttestationPoolResponse{
-				NextPageToken: "1",
```

Comment:
I am not sure why this test expects to receive a NextPageToken here since its also requesting all items in one page. Maybe I am misinterpreting the api how it is working right now.